apps/docs/src/pages/components/tree-view.mdx

```diff
@@ -0,0 +1,433 @@
+import { Callout } from "nextra/components";
+import {
+  HvPanel,
+  HvTreeItem,
+  HvTreeView,
+  treeView2Classes,
+} from "@hitachivantara/uikit-react-core";
+
+import Playground from "@docs/components/code/Playground";
+import { Description } from "@docs/components/page/Description";
+import { Page } from "@docs/components/page/Page";
+import { getComponentData } from "@docs/utils/component";
+
+export const getStaticProps = async ({ params }) => {
+  const meta = await getComponentData("TreeView", "core", treeView2Classes, [
+    "TreeItem",
+  ]);
+  return { props: { ssg: { meta } } };
+};
+
+<Description />
+
+<Page>
+
+<Playground
+  Component={HvTreeView}
+  componentName="HvTreeView"
+  controls={{
+    multiSelect: { defaultValue: false },
+    disableSelection: { defaultValue: false },
+}}
+decorator={(component) => (
+<HvPanel style={{ width: 300 }}>{component}</HvPanel>
+)}
+
```

Comment:
there must be errors somewhere in the samples, because the code isn't formatted correctly

Reply:
I couldn't find any formatted the code properly.